rs/execution_environment/src/execution_environment.rs

```diff
@@ -3121,7 +3174,6 @@ impl ExecutionEnvironment {
                         Err(err.into())
                     }
                 };
-                canister.update_on_low_wasm_memory_hook_condition();
```

Comment:
No longer required because we now check condition after every mgmt canister call.